.github/workflows/beaver_api.yml

```diff
@@ -0,0 +1,97 @@
+name: Beaver API CI
+
+on:
+  workflow_call:
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    env:
+      DEBUG: True
+      SECRET_KEY: ${{ secrets.SECRET_KEY }}
```

Comment:
I think you should use `with` context to access secrets [the doc](https://docs.github.com/en/actions/security-guides/using-secrets-in-github-actions#using-secrets-in-a-workflow)